src/components/media_manager/src/socket_streamer_adapter.cc

```diff
@@ -70,6 +70,8 @@ SocketStreamerAdapter::SocketStreamer::SocketStreamer(
     ip_(ip),
     port_(port),
     header_(header),
+    server_socket_(),
```

Comment:
It's considered as "best practice" to init all members in the correct order in the ctor. I don't quite remember who made such advice. Seems it was Meyers or McConnel.